examples/ollama/README.md

```diff
@@ -0,0 +1,36 @@
+## Ollama example
+
+This example demonstrates how to run Ollama (UI and api) using helm on Control Plane.
```

Comment:
The user interface is the project https://github.com/open-webui/open-webui
It runs on port 8080 as a sidecar to the ollama API. Since 8080 is the first port specified in the workload definition all external traffic is forwarded to it.

The ollama API is the project https://github.com/ollama/ollama
It runs on port 11434 and is accessed by the open-webui sidecar. There is a persistent storage volume of 10Gib (default) that is used to store the models. On startup, a script is used to download a default model (default llama2) if it does not yet exist on the filesystem.

On Control Plane, you can access T4 GPU's from any cloud provider. You can even deploy this example to multiple cloud provider geo locations at the same time and end users will be routed to the closest available location.